.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+## Issue Overview
```

Comment:
Probably above this should be a link [CONTRIBUTING.md](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md)